pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -76,6 +76,7 @@ <h2 class="my-4 font-bold">2. ORIGINATOR DETAILS</h2>
         {{ end }}
 
         {{ $originatorBirth := $originator.DateAndPlaceOfBirth }}
+        {{ if $originatorBirth }}
```

Comment:
The conditional was added to handle transactions created before `Date of Birth` and `Place of Birth` were added to the response to fix an issue that prevented the entire form from rendering.